### PR TITLE
18495 - Coop Correction Changes when the Resolution is Absent

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:* [/bcgov/entity18495](https://github.com/bcgov/entity/issues/18495)

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-edit-dev--pr-551-vq346fsh.web.app

**Comment by seeker25** [comment]
Need to change this a bit:

https://github.com/bcgov/lear/blob/75c7c6f925775e24a13679f429903e477c8ebfbf/queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/correction.py#L131

File: src/components/SpecialResolution/Memorandum.vue [hunk 1]
```diff
@@ -117,8 +118,18 @@
...
                   </template>
                 </v-checkbox>
+                <span
+                  v-if="!hasResolutionOnFile"
+                  class="black-bold-font"
+                >
+                  Upload a new full set of the memorandum PDF document
+                </span>
+                <v-spacer class="spacer" />
+                <UploadRules
```
Comment (seeker25) [inline src/components/SpecialResolution/Memorandum.vue:128]: Probably should rename this component, `UploadRulesOrMemorandum` or make it generic?

**Review by seeker25 — APPROVED** [review]

File: src/store/store.ts [hunk 2]
```diff
@@ -1291,7 +1296,9 @@ export const useStore = defineStore('store', {
     },
 
     hasSpecialResolutionMemorandumChanged (): boolean {
-      return this.getSpecialResolutionMemorandum?.includedInResolution
+      return this.getSpecialResolutionMemorandum?.includedInResolution ||
+      (this.getSpecialResolutionMemorandum?.key || null) !==
```
Comment (JazzarKarim) [inline src/store/store.ts:1300]: Jia, please if you can, add extra indentation here and the line below.

**Review by JazzarKarim — APPROVED** [review]
Looks great Jia! 👍

**Pull request merged**